src/components/Upload/Upload.types.tsx

```diff
@@ -249,6 +249,11 @@ export interface ShowUploadListInterface {
    * @default 'button'
    */
   downloadIconButtonType?: UploadButtonHtmlType;
+  /**
+   * Assigns Upload list 100% width.
+   * @default false
+   */
+  fullWidth?: boolean;
```

Comment:
noticed in your attached video that the error item in the list was not full width like the others.

Reply:
does the user have to specify this or is it fully controlled based on the parent component implementation?

Reply:
No, I missed it in css, update pushed. :)